src/main/java/org/cloudfoundry/community/servicebroker/s3/service/S3ServiceInstanceBindingService.java

```diff
@@ -15,65 +15,44 @@
  */
 package org.cloudfoundry.community.servicebroker.s3.service;
 
-import java.util.HashMap;
-import java.util.Map;
-
 import org.cloudfoundry.community.servicebroker.exception.ServiceBrokerException;
 import org.cloudfoundry.community.servicebroker.exception.ServiceInstanceBindingExistsException;
 import org.cloudfoundry.community.servicebroker.model.ServiceInstance;
 import org.cloudfoundry.community.servicebroker.model.ServiceInstanceBinding;
+import org.cloudfoundry.community.servicebroker.s3.plan.BasicPlan;
 import org.cloudfoundry.community.servicebroker.service.ServiceInstanceBindingService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
-import com.amazonaws.services.identitymanagement.model.AccessKey;
-import com.amazonaws.services.identitymanagement.model.User;
-
 /**
  * @author David Ehringer
  */
 @Service
 public class S3ServiceInstanceBindingService implements ServiceInstanceBindingService {
-
-    private final S3 s3;
-    private final Iam iam;
+    private final BasicPlan basicPlan;
```

Comment:
Thanks! Instead we've created a Plan interface in commit https://github.com/mendix/s3-cf-service-broker/commit/cf4a4e7